holmes/core/tools.py

```diff
@@ -323,7 +301,10 @@ class ToolsetEnvironmentPrerequisite(BaseModel):
 class Toolset(BaseModel):
     model_config = ConfigDict(extra="forbid")
     experimental: bool = False
-    enabled: bool = False
+
+    # Built-in toolset is enabled when 1. it's specified in the config and 2. 'enabled' is not set to false
+    # Custom toolset is enabled when 'enabled' is not set to false
+    enabled: bool = True
```

Comment:
This change causing to scenario when we are trying to load a lot of irrelevant tools in the server and most of the in error state. 
![image](https://github.com/user-attachments/assets/37224b6f-d902-43e4-a3d4-303a0ae15766)

We need to make sure that in server it will be as today.
![image](https://github.com/user-attachments/assets/244f524c-dfb7-4b77-a844-b7baf380f738)